src/frontend/js/modules/esn.summernote.spec.js

```diff
@@ -10,15 +10,16 @@ describe('The esn.summernote Angular module', function() {
   esnI18nServiceMock = {
     getFullLocale: sinon.spy(function(cb) {
       return cb(fullLocale || ESN_I18N_DEFAULT_FULL_LOCALE);
-    })
+    }),
+    getLocale: function() { return "fr" }
```

Comment:
Why do you have to add this, and:

```suggestion
    getLocale: function() { return 'fr'; }
```